exaudfclient/base/javacontainer/test/cpp/javacontainer_perf_test.cc

```diff
@@ -24,4 +24,35 @@ TEST(JavaContainerPerformance, large_inline_java_udf_test) {
     JavaVMTest vm(script_code);
 }
 
+TEST(JavaContainerPerformance, large_inline_single_line_full_java_udf_test) {
+    std::string script_code =
+        "%jvmoption option1=abc;"
+        "%jvmoption option2=def;"
+        "class JVMOPTION_TEST_WITH_SPACE {"
+        "static void run(ExaMetadata exa, ExaIterator ctx) throws Exception {";
+
+    for (uint32_t idxLine(0); idxLine < NumInlineJavaLines; idxLine++) {
```

Comment:
```suggestion
    for (uint32_t idxLine(0); idxLine < NumInlineJavaLines; idxLine++) {
```
Why did you choose `idxLine(0)` over `idxLine = 0`?

Reply:
This is just a habit. AFIK there is no difference between both ways....